src/main/java/vecrosen/CustomDateTimeParser.java

```diff
@@ -0,0 +1,38 @@
+package vecrosen;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Class containing static function to parse dates.
+ */
+public class CustomDateTimeParser {
+    private static final String[] supportedDates = {
+            "uuuu-MM-dd", "uu-MM-dd",
+            "dd MMM, uuuu", "MMM dd, uuuu", "dd MMM uuuu", "MMM dd uuuu", "uuuu MMM dd", "uuuu dd MMM",
+            "dd MMMM, uuuu", "MMMM dd, uuuu", "dd MMMM uuuu", "MMMM dd uuuu", "uuuu MMMM dd", "uuuu dd MMMM",
+            "MM/dd/uuuu", "MM/dd/uu", "MM/dd",
+            "MMM dd", "dd MMM",
+            "MMMM dd", "dd MMMM"
```

Comment:
Should the indentation be at 8 instead of 12?

Reply:
I figured it should be at 12 since this is a wrapped line, even though the parent line ended in a {.